### DISCUSSION TITLE
LANL Workshop

### DISCUSSION THREAD

**Discussion by djm87** [lanl/MILK#3] [Announcements]
This release comes at the end of Luca's visit at LANL and takes advantages of the changes to MAUD. I checked it works with 2.996 on Windows, Linux, and Mac. I did not test the ARM build for Mac. 

<hr /><em>This discussion was created from the release <a href='https://github.com/lanl/MILK/releases/tag/v0.2'>LANL Workshop</a>.</em>

**Comment by johnokie** [comment]
Using Windows.... typos in script for example #3: 
starting 2nd python script  "python 2_setups.py"  the file is missing the "s" at the end.

FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\**\\cinema_debye_scherre' 
   missing the last "r" in debye_scherrer

**Reply by ZhangxiJesseFeng** [reply]
Thanks John I fixed the typos.
